rules/library.bzl

```diff
@@ -219,6 +220,82 @@ def _canonicalize_swift_version(swift_version):
 
     return swift_version
 
+def _xcframework_build_type(*, linkage, packaging):
+    return (linkage, packaging)
+
+def _xcframework_slice(*, xcframework_name, identifier, platform, platform_variant, supported_archs, build_type, path):
+    linkage, packaging = _xcframework_build_type(**build_type)
+    import_name = "{}-{}".format(xcframework_name, identifier)
+    if (linkage, packaging) == ("dynamic", "framework"):
+        apple_dynamic_framework_import(
```

Comment:
See lower comment, isn't this a multi-arch or mutli platform handling rule?

Reply:
it is multi-arch, and indeed we're allowing it to handle slices that have multiple archs